app/models/user.rb

```diff
@@ -0,0 +1,27 @@
+require 'data_mapper'
+require 'bcrypt'
+require 'pry'
+
+class User
+  include DataMapper::Resource
+
+  attr_reader   :password
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.